modules/virt-hardware-os-requirements.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+:_content-type: REFERENCE
+[id="virt-hardware-os-requirements_{context}"]
+= Hardware and operating system requirements
+
+Review the following hardware and operating system requirements for {VirtProductName}.
+
+.Supported platforms
+* On-premise bare metal servers
+* Amazon Web Services bare metal instances
+* IBM Cloud bare metal servers. See link:https://access.redhat.com/articles/6738731[Deploy OpenShift Virtualization on IBM Cloud Bare Metal nodes].
+
+ifdef::openshift-enterprise[]
+:FeatureName: Installing OpenShift Virtualization on AWS bare metal instances or on IBM Cloud bare metal servers
+include::snippets/technology-preview.adoc[]
+:!FeatureName:
+endif::[]
+
+Bare metal instances or servers offered by other cloud providers are not supported.
```

Comment:
Question: What is the difference between servers and instances in this context?

Reply:
No difference. However, AWS calls them "instances" and IBM Cloud calls them "servers". If I used only "servers", someone might actually think that they could use "instances" from another cloud provider. Or vice versa, if I used "instances". I'm just covering my bases.